packages/presentation/lib/ui/auth_page/auth_widget.dart

```diff
@@ -0,0 +1,287 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:presentation/base/bloc_screen.dart';
+import 'package:presentation/base/tile_wrapper.dart';
+import 'package:presentation/colors_application/colors_application.dart';
+import 'package:presentation/generated/l10n.dart';
+import 'package:presentation/library/dimens/dimens.dart';
+import 'package:presentation/library/images_utils/images_utils.dart';
+import 'package:presentation/library/style/text_style.dart';
+import 'package:presentation/navigation/base_page.dart';
+import 'package:presentation/ui/auth_page/bloc/auth_bloc.dart';
+import 'package:presentation/ui/auth_page/bloc/auth_tile.dart';
+
+class AuthWidget extends StatefulWidget {
+  const AuthWidget({super.key});
+
+  static const _routeName = '/AuthWidget';
+
+  static BasePage page() => BasePage(
+        key: const ValueKey(_routeName),
+        name: _routeName,
+        builder: (context) => const AuthWidget(),
+        showSlideAnim: true,
+        showBottomBar: true,
+      );
+
+  @override
+  State createState() => _AuthWidgetState();
+}
+
+class _AuthWidgetState extends BlocScreenState<AuthWidget, AuthBloc> {
+  @override
+  Widget build(BuildContext context) {
+    return StreamBuilder<TileWrapper<AuthTile>>(
+      stream: bloc.dataStream,
+      builder: (_, snapshot) {
+        final data = snapshot.data;
+        final tile = data?.data;
+        return Scaffold(
+          backgroundColor: ColorsApplication.colorTheme,
+          appBar: AppBar(
+            bottom: PreferredSize(
+              preferredSize: const Size.fromHeight(Dimens.size1),
+              child: Container(
+                color: ColorsApplication.colorBorder,
+                height: Dimens.size1,
+              ),
+            ),
```

Comment:
seems a bit complicated. cannot we use only Container, without PreferredSize?

Reply:
<img width="337" alt="image" src="https://user-images.githubusercontent.com/111364799/194411709-50e5ee01-2f03-4b4f-a631-b3a174cd6770.png">
unfortunately(